Lib/importlib/_bootstrap.py

```diff
@@ -1437,6 +1551,8 @@ def _setup(sys_module, _imp_module):
             builtin_module = sys.modules[builtin_name]
         setattr(self_module, builtin_name, builtin_module)
 
+    _blocking_on = WeakValueDictionary()
```

Comment:
I suppose that it cannot be created until _weakref is available. Would you mind to add a comment explaining that? It wasn't obvious to me when I first read the code.